doc/man3/SSL_CTX_set_min_proto_version.pod

```diff
@@ -29,21 +35,27 @@ Setting the minimum or maximum version to 0, will enable protocol
 versions down to the lowest version, or up to the highest version
 supported by the library, respectively.
 
+Getters return 0 in case B<ctx> or B<ssl> have been configured to
+automatically use the lowest or highest version supported by the library.
+
 Currently supported versions are B<SSL3_VERSION>, B<TLS1_VERSION>,
 B<TLS1_1_VERSION>, B<TLS1_2_VERSION>, B<TLS1_3_VERSION> for TLS and
 B<DTLS1_VERSION>, B<DTLS1_2_VERSION> for DTLS.
 
 =head1 RETURN VALUES
 
-These functions return 1 on success and 0 on failure.
+These setter functions return 1 on success and 0 on failure. The getter
+functions return the configured version or 0 for auto-configuration of
```

Comment:
Do we want to give an example and/or pointer to relevant header for what these values would look like (e.g., TLS1_2_VERSION)?

Reply:
We list the versions a couple lines above. I guess it *should* be obvious to any reader, don't you think?

Reply:
Yes, it should be; thanks for pointing it out